### ISSUE TITLE
About the results on the ImageNet VID validation set(TransVOD)

### ISSUE THREAD

**Issue by Video-Object-Detection** [SJTU-LuHe/TransVOD#48]
Thank you very much for your work. I followed the same experimental setup and tested using the release pre-trained model but could not achieve the same experimental results. I only used two GPU for testing, other settings are the same as the original code.
Is there any other setting that needs to be changed?

Thank you for your reply.

![Image](https://github.com/user-attachments/assets/33f9d7ce-d056-4365-a2a3-386aef96236e)

TransVOD-R50(79.9)

![Image](https://github.com/user-attachments/assets/08560ca1-d980-4c0c-a75e-3fab5a459204)

TransVOD-R101(81.9)

![Image](https://github.com/user-attachments/assets/4a425164-9809-4a9b-8cc6-06399db27aa1)
